Cargo.toml

```diff
@@ -62,31 +62,31 @@ bytes = "1.5.0"
 rcgen = { version = "0.10.0", features = ["pem", "x509-parser"] }
 mime = "0.3.17"
 aws-sdk-s3 = { version = "1.36.0", features = [
-  "behavior-version-latest",
-  "rt-tokio",
+    "behavior-version-latest",
+    "rt-tokio",
 ] }
 aws-config = { version = "1.5.1", features = ["behavior-version-latest"] }
 redis = { workspace = true, features = [
-  "json",
-  "tokio-comp",
-  "connection-manager",
+    "json",
+    "tokio-comp",
+    "connection-manager",
 ] }
 tracing = { version = "0.1.40", features = ["log"] }
 tracing-subscriber = { version = "0.3.18", features = [
-  "registry",
-  "env-filter",
-  "ansi",
-  "json",
-  "tracing-log",
+    "registry",
+    "env-filter",
+    "ansi",
+    "json",
+    "tracing-log",
 ] }
 tracing-bunyan-formatter = "0.3.9"
 sqlx = { workspace = true, default-features = false, features = [
-  "runtime-tokio-rustls",
-  "macros",
-  "postgres",
-  "uuid",
-  "chrono",
-  "migrate",
+    "runtime-tokio-rustls",
```

Comment:
In order to make it consistency. You can modify your IDE setting as shown below
<img width="1109" alt="image" src="https://github.com/user-attachments/assets/56e0d1ff-8d2d-4c02-98c9-bb86bab3a178">